project_depatment/application/templates/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
templates is not a package - init.py is redundant

Reply:
fixed